price-estimator/src/main.rs

```diff
@@ -58,6 +55,11 @@ struct Options {
     #[structopt(long, env = "ETHEREUM_NODE_URL")]
     node_url: Url,
 
+    /// The network ID used for gas estimations.
+    /// For example 1 for mainnet, 4 for rinkeby.
+    #[structopt(long, env = "NETWORK_ID", default_value = "1")]
```

Comment:
Not related to this PR but I think we should be able to get the network ID from the ethereum node URL with `eth_chainId` or `eth_networkId` call.